tcms/testruns/static/testruns/js/get.js

```diff
@@ -282,6 +294,35 @@ function renderTestExecutionRow(testExecution) {
     template.find('.test-execution-status-icon').addClass(testExecutionStatus.icon).css('color', testExecutionStatus.color)
     template.find('.test-execution-status-name').html(testExecutionStatus.name).css('color', testExecutionStatus.color)
 
+    const commentField = template.find('.comment-text')[0]
+    const fileInputField = template.find('.simplemde-file-upload')
+    const mdeCommentForm = initSimpleMDE(commentField, fileInputField, `comment-form-for-te-${testExecution.id}`)
+    const commentsRow = template.find('.comments')
+
+    template.find('.cancel-comment').click(() => mdeCommentForm.value(''))
+    template.find('.post-comment').click(() => {
+        const input = mdeCommentForm.value().trim()
+
+        if (input) {
+            jsonRPC('TestExecution.add_comment', [testExecution.id, input], comment => {
+                mdeCommentForm.value('')
+                const testExecutionRow = $(`.test-execution-${testExecution.id}`)
```

Comment:
isn't that referenced by the `template` variable already ?

Reply:
not the same, but we achieve the same result with `template.find` instead of `testExecutionRow.find`